account_customer_wallet/models/res_partner.py

```diff
@@ -19,6 +19,9 @@ class Partner(models.Model):
         readonly=True,
         search="_search_customer_wallet_balance",
     )
+    is_customer_wallet_user = fields.Boolean(
+        string="Is Customer Wallet User",
+    )
```

Comment:
Should this not be a computed field? If there are one or more lines in the journal/account containing the (family of the) user, then this should be True, else False.